### PR TITLE
updated config

### PR DESCRIPTION
**Pull request by Aaronstar-McClure**
updated config, please check. ran through jsonlint and is valid.

### CODE REVIEW

**Comment by Aaronstar-McClure** [comment]
also, there is no CONTRIBUTING.md file

File: config/config.json.example [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+    ],
+    "bypass": {
+        "roles": [
+            "ATLauncher Staff",
+            "NodeCraft Staff",
+            "Moderators",
+            "AllmightyBot"
+        ],
+        "users": [
+		]
```
Comment (RyanTheAllmighty) [inline config/config.json.example]: This looks to be indented one too short

File: config/config.json.example [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
+    "non_support_channels": [
+        "general"
+    ],
+    "moderated_channels": [
+        "general",
+        "support"
+    ],
+    "bypass": {
+        "roles": [
+            "ATLauncher Staff",
```
Comment (RyanTheAllmighty) [inline config/config.json.example]: Can we change this to simply just include one role with a name of "Staff" just as an example so it's not 'ATLauncher' specific

**Review by RyanTheAllmighty — CHANGES_REQUESTED** [review]
Just a couple minor changes

File: config/config.json.example (hunk 1 above)
Comment (Aaronstar-McClure) [reply config/config.json.example]: users is under bypass next to roles, their square brackets are parallel. I believe it is correct. jsonlint finds no problem with this

File: config/config.json.example (hunk 2 above)
Comment (Aaronstar-McClure) [reply config/config.json.example]: changed to owner and moderator to indicate comma list. jsonlint cleared errors

**Comment by Aaronstar-McClure** [comment]
changes applied

File: config/config.json.example (hunk 1 above)
Comment (Aaronstar-McClure) [reply config/config.json.example]: correction: when viewed in the online editor it shows as aligned, however in the viewer, it is one off. will investigate

Comment (Aaronstar-McClure) [reply config/config.json.example]: was two tabs, is now spaces

File: config/config.json.example [hunk 3]
```diff
@@ -1,16 +1,28 @@
...
+    "moderated_channels": [
+        "general",
+        "support"
+    ],
+    "bypass": {
+        "roles": [
+            "Owner",
+            "Moderator"
+        ],
+        "users": [
```
Comment (RyanTheAllmighty) [inline config/config.json.example]: Last one I promise. When empty, json arrays should be on the same line:

```
{
    "users": []
}
```

**Review by RyanTheAllmighty — CHANGES_REQUESTED** [review]

**Comment by RyanTheAllmighty** [comment]
Thanks again for this @Aaronstar-McClure, sorry it took so long, just been MIA for the last month, been super busy. All cleared up now so I really do appreciate this

**Comment by Aaronstar-McClure** [comment]
Done

**Review by RyanTheAllmighty — APPROVED** [review]

**Pull request merged**